src/components/FarmDualCard/FarmDualCard.tsx

```diff
@@ -131,120 +129,108 @@ const FarmDualCard: React.FC<{
+              <Typography variant='body2'>{tvl}</Typography>
             </Box>
-          </Box>
-        </Box>
-        <Box
-          width={isMobile ? 1 : 0.2}
-          display='flex'
-          justifyContent={isMobile ? 'space-between' : 'flex-end'}
-        >
-          {isMobile && (
-            <Typography className={classes.farmDualText}>Earned</Typography>
-          )}
-          <Box textAlign='right'>
-            <Typography variant='body2'>{earnedUSDStr}</Typography>
-            <Box display='flex' alignItems='center' justifyContent='flex-end'>
-              <CurrencyLogo
-                currency={returnTokenFromKey('QUICK')}
-                size='16px'
-              />
-              <Typography variant='body2' style={{ marginLeft: 5 }}>
-                {stakingInfo.earnedAmountA.toSignificant(2)}
-                <span>&nbsp;dQUICK</span>
+            <Box width={0.25} textAlign='center'>
+              <Typography variant='body2'>
+                ${Number(rewards.toFixed(0)).toLocaleString()} / day
               </Typography>
+              <Typography variant='body2'>{poolRateA}</Typography>
+              <Typography variant='body2'>{poolRateB}</Typography>
             </Box>
-            <Box display='flex' alignItems='center' justifyContent='flex-end'>
-              <CurrencyLogo
-                currency={
-                  rewardTokenB.symbol?.toLowerCase() === 'wmatic'
-                    ? ETHER
-                    : rewardTokenB
-                }
-                size='16px'
-              />
-              <Typography variant='body2' style={{ marginLeft: 5 }}>
-                {stakingInfo.earnedAmountB.toSignificant(2)}
-                <span>&nbsp;{rewardTokenB.symbol}</span>
-              </Typography>
+            <Box
+              width={0.15}
+              display='flex'
+              justifyContent='center'
+              alignItems='center'
+              color={palette.success.main}
+            >
+              <Typography variant='body2'>{apyWithFee}%</Typography>
+              <Box ml={0.5} height={16}>
+                <img src={CircleInfoIcon} alt={'arrow up'} />
+              </Box>
+            </Box>
+            <Box width={0.2} textAlign='right'>
+              <Typography variant='body2'>{earnedUSDStr}</Typography>
+              <Box display='flex' alignItems='center' justifyContent='flex-end'>
+                <CurrencyLogo
+                  currency={returnTokenFromKey('QUICK')}
+                  size='16px'
+                />
+                <Typography variant='body2' style={{ marginLeft: 5 }}>
+                  {stakingInfo.earnedAmountA.toSignificant(2)}
+                  <span>&nbsp;dQUICK</span>
+                </Typography>
+              </Box>
+              <Box display='flex' alignItems='center' justifyContent='flex-end'>
+                <CurrencyLogo
+                  currency={
+                    rewardTokenB.symbol?.toLowerCase() === 'wmatic'
```

Comment:
this check is done in 7 different places in the code, this should be a function

`rewardTokenB.symbol?.toLowerCase() === 'wmatic' ? ETHER : rewardTokenB